php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -456,12 +456,16 @@ public FunctionDeclarationTree functionDeclaration(
 
   public ParameterListTree parameterList(
     InternalSyntaxToken leftParenthesis,
-    Optional<Tuple<ParameterTree, Optional<List<Tuple<InternalSyntaxToken, ParameterTree>>>>> parameters,
+    Optional<Tuple<Tuple<ParameterTree, Optional<List<Tuple<InternalSyntaxToken, ParameterTree>>>>, Optional<InternalSyntaxToken>>> parameters,
     InternalSyntaxToken rightParenthesis
   ) {
     SeparatedListImpl<ParameterTree> separatedList = SeparatedListImpl.empty();
     if (parameters.isPresent()) {
-      separatedList = separatedList(parameters.get().first(), parameters.get().second());
+      if (parameters.get().second().isPresent()) {
+        separatedList = separatedList(parameters.get().first().first(), parameters.get().first().second(), parameters.get().second().orNull());
```

Comment:
Small thing, but I think the `orNull()` is redundant as you have already checked for presence. a direct `get()` should be fine.

Reply:
Now that you make me notice that, I can leave the orNull() call, and remove the check for presence.